Software-Engineering/L1.0-Graduate-Software-Engineer.md

```diff
@@ -2,7 +2,7 @@
 
 This position is an entry level role, within Level 1. 
 The expectations of a Graduate Software Engineer are equivalent to a [Junior Software Engineer](L1-Junior-Software-Engineer.md), except with no assumed prior knowledge.
-Career progression from this role is possible via the subsequent bands as a Junior Software Engineer:

```

Comment:
I assume that you're proposing this change just to bring it in line with the language used in the performance review form, which I understand entirely :shipit:

I see some subtle disadvantages of using "maturities" rather than "band", but feel free to disregard if you want to exclude these considerations from the scope of your change:

1. One of the "maturities" would be `Maturing`, which seems awkward naming
2. We might accidentally send the impression that moving from `Performing` to `Exceeding` is associated with further "maturing" into that level, which is not the case (rather, someone who has "fully matured" into a level will be at `Performing`).

Hope that helps 🙇

Reply:
>just so that we use consistent language across here and the perf review form
Yep. One step in a consistent direction for now.